snapshot/pkg/volume/openebs/processor.go

```diff
@@ -69,15 +78,22 @@ func (h *openEBSPlugin) SnapshotCreate(snapshot *crdv1.VolumeSnapshot, pv *v1.Pe
 
 	snapObj := (*tags)["kubernetes.io/created-for/snapshot/name"]
 	snapshotName := createSnapshotName(pv.Name, snapObj)
+
 	casType := pv.Annotations["openebs.io/cas-type"]
 	if casType == "" {
 		casType = "jiva"
 	}
+
+	ok := supportedCasType[casType]
+	if !ok {
+		return nil, nil, fmt.Errorf("aborting snapshot create operation as specified cas-Type '%s' which is not suported", casType)
```

Comment:
_n: `aborting create snapshot operation as specified volume type (%s) does not support snapshots`

Reply:
done